C/include/c4Index.h

```diff
@@ -183,6 +194,21 @@ CBL_CORE_API bool c4indexupdater_finish(C4IndexUpdater* updater, C4Error* outErr
 
 #endif
 
+
+#ifndef C4_STRICT_COLLECTION_API
+//======== SEMI-DEPRECATED DATABASE METHODS:
+NODISCARD CBL_CORE_API bool c4db_createIndex(C4Database* database, C4String name, C4String indexSpecJSON,
```

Comment:
.NET, at least, doesn't use these anymore so it might be time to actually deprecate, or remove these functions (c4coll_isIndexTrained can wait a bit maybe but since it was JUST introduced it should only be present in a few places).

Reply:
Sounds good -- could you file an issue?